auto_tests/file_transfer_test.c

```diff
@@ -119,27 +105,19 @@ static bool file_sending_done;
 static void tox_file_chunk_request(Tox *tox, uint32_t friend_number, uint32_t file_number, uint64_t position,
                                    size_t length, void *user_data)
 {
-    if (!sendf_ok) {
-        ck_abort_msg("Didn't get resume control");
-    }
+    ck_assert_msg(sendf_ok, "didn't get resume control");
 
-    if (sending_pos != position) {
-        ck_abort_msg("Bad position %llu", (unsigned long long)position);
-    }
+    ck_assert_msg(sending_pos == position, "bad position %llu", (unsigned long long) position);
```

Comment:
`%llu` is not supported on windows. Change to `%lu` and cast to `unsigned long`.